kong/pdk/node.lua

```diff
@@ -147,6 +161,17 @@ local function new(self)
   --       pid = 18005
   --     }
   --   }
+  --   -- if the `kong` use dbless mode, the following will be present:
+  --  lmdb = {
+  --    max_map_size: "131072 KB",
+  --    map_size: "131072 KB",
+  --    used_size": "20.48 KB",
+  --    used_pages": 6,
+  --    max_pages": 32768,
+  --    last_txnid": 2,
+  --    max_readers": 126,
+  --    num_readers": 16
```

Comment:
Is this the number of current readers? If yes, would current_readers make more sense?